Dockerfile

```diff
@@ -6,18 +6,11 @@ COPY . .
 
 RUN yarn add shx -W --ignore-scripts && \
     yarn install --ignore-scripts && \
+    yarn playwright install --with-deps && \
     yarn build
 
-###
-
-FROM cgr.dev/chainguard/node:latest
-
-WORKDIR /app
-
-COPY --chown=node:node --from=builder /app ./
```

Comment:
Did you try to keep this multi-stage version and to install playwright?
`RUN yarn playwright install --with-deps`

Reply:
yes it was what I try first but it didn't worked.  
In my memories the error was because `playwright install --with-deps` needs `bash`